src/jit/gtlist.h

```diff
@@ -161,7 +161,7 @@ GTNODE(ADD_LO     , "+Lo"          ,1,GTK_BINOP)
 GTNODE(ADD_HI     , "+Hi"          ,1,GTK_BINOP)
 GTNODE(SUB_LO     , "-Lo"          ,0,GTK_BINOP)
 GTNODE(SUB_HI     , "-Hi"          ,0,GTK_BINOP)
-GTNODE(MUL_HI     , "*Hi"          ,1,GTK_BINOP)
+GTNODE(MUL_HI     , "*Hi"          ,1,GTK_BINOP) // A mul that returns the 2N bit result of an NxN multiply
```

Comment:
I'm under the impression that if I use a different name, MUL_HI becomes obsolete, and can be removed. Does that sound accurate to you?